packages/framework-provider-local-infrastructure/src/test-helper/local-test-helper.ts

```diff
@@ -0,0 +1,43 @@
+import { LocalQueries } from './local-queries'
+import { LocalCounters } from './local-counters'
+
+interface ApplicationOutputs {
+    graphqlURL: string
+    websocketURL: string
+}
+
+export class LocalTestHelper {
+    private constructor(
+        readonly outputs: ApplicationOutputs,
+        readonly counters: LocalCounters,
+        readonly queries: LocalQueries
+    ) {}
```

Comment:
class used by integration tests. It follows the mandatory interface for those tests